src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -129,22 +138,27 @@ public double getClimberVelocity() {
     public Command raiseClimber() {
         return Commands.startEnd(
                 () -> {
-                    setTargetVoltage(-7);
```

Comment:
any reason to change the speeds to be slower? it'll make the robot take more time to climb!